components/script/dom/webglprogram.rs

```diff
@@ -23,41 +22,96 @@ use dom_struct::dom_struct;
 use fnv::FnvHashSet;
 use std::cell::Cell;
 
+struct DroppableField {
+    id: WebGLProgramId,
+    marked_for_deletion: Cell<bool>,
+    is_in_use: Cell<bool>,
+    fragment_shader: MutNullableDom<WebGLShader>,
+    vertex_shader: MutNullableDom<WebGLShader>,
+    context: WebGLRenderingContext,
```

Comment:
Same as previous - let's use the sender directly instead of the context.

Reply:
I fixed. https://github.com/servo/servo/pull/28302/commits/ce4fe085a703b299391ae746ac7d315d055d3b84